src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
+
+
     public String getRandomColor() {
-        return null;
+
+        int colors = new Random().nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[colors]);
```

Comment:
Use the 'name()' method instead of 'String.valueOf()' when you need a String representation of an enum constant. It's clearer and more direct. Replace 'String.valueOf(Colors.values()[colors])' with 'Colors.values()[colors].name()'.